app/routes/project/messages/new.js

```diff
@@ -0,0 +1,66 @@
+import { get } from '@ember/object';
+import { inject as service } from '@ember/service';
+import Route from '@ember/routing/route';
+import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
+import { CanMixin } from 'ember-can';
+import { hash } from 'rsvp';
+
+export default Route.extend(AuthenticatedRouteMixin, CanMixin, {
+  currentUser: service(),
+  store: service(),
+
+  queryParams: { userId: { as: 'user', refreshModel: true } },
+
+  async model({ userId }) {
+    let store = get(this, 'store');
+    let project = this.modelFor('project');
+
+    let { user, currentUser } = await hash({
+      user: store.find('user', userId),
+      currentUser: get(this, 'currentUser.user')
+    });
+
+    let message =  store.createRecord('message', {
+      author: currentUser,
+      initiatedBy: 'admin',
+      project
+    });
+
+    let conversation = store.createRecord('conversation', { user });
+    get(message, 'conversations').pushObject(conversation);
```

Comment:
This causes the conversation to appear in the list of conversations in the sidebar.